run_all_tests.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 export SBT_OPTS="-XX:+CMSClassUnloadingEnabled -Xmx4G"
-sbt pre-commit
+sbt -Dwebdriver.chrome.driver=/usr/bin/chromedriver pre-commit
```

Comment:
Is this intentionally using the local driver?

Reply:
No - I'm confused it's still there as I worked out how to get it working using JAVA_TOOLS_OPTIONS so this was removed.  Maybe a late local commit that was not pushed.